.github/workflows/release.yml

```diff
@@ -0,0 +1,577 @@
+name: Release
```

Comment:
Commit message should start with "ci". For example ci: publish suspicious cases nuget